src/test/ui/span/coerce-suggestions.stderr

```diff
@@ -38,9 +38,12 @@ error[E0308]: mismatched types
   --> $DIR/coerce-suggestions.rs:17:9
    |
 LL |     f = Box::new(f);
-   |         ^^^^^^^^^^^- help: try using a conversion method: `.to_string()`
-   |         |
-   |         cyclic type of infinite size
+   |         ^^^^^^^^^^^ cyclic type of infinite size
+   |
+help: consider unboxing the value
+   |
+LL |     f = *Box::new(f);
+   |         +
```

Comment:
This suggestion is kind of funny, but I guess it's fine because of the pre-existing `to_string` suggestion.

Reply:
yeah the other suggestions are already kinda wacky anyways lol